<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDebug</class>
 <widget class="QWidget" name="FormDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QDockWidget" name="dockWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>402</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>524287</height>
    </size>
   </property>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="tbTab1">
       <property name="toolTip">
        <string>显微镜设置</string>
       </property>
       <property name="statusTip">
        <string>显微镜设置</string>
       </property>
       <property name="text">
        <string>显微镜设置</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/micro.png</normaloff>:/icon/micro.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTab2">
       <property name="toolTip">
        <string>性状摄像调整</string>
       </property>
       <property name="statusTip">
        <string>性状摄像调整</string>
       </property>
       <property name="text">
        <string>性状摄像调整</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/ccd.png</normaloff>:/icon/ccd.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTab3">
       <property name="toolTip">
        <string>干化摄像调整</string>
       </property>
       <property name="statusTip">
        <string>干化摄像调整</string>
       </property>
       <property name="text">
        <string>干化摄像调整</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/ccd.png</normaloff>:/icon/ccd.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTab4">
       <property name="toolTip">
        <string>控制参数设置</string>
       </property>
       <property name="statusTip">
        <string>控制参数设置</string>
       </property>
       <property name="text">
        <string>控制参数设置</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/setcolor.png</normaloff>:/icon/setcolor.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTab5">
       <property name="toolTip">
        <string>质量控制</string>
       </property>
       <property name="statusTip">
        <string>质量控制</string>
       </property>
       <property name="text">
        <string>质量控制</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/qc.png</normaloff>:/icon/qc.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMdiArea" name="mdiArea">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>1</y>
     <width>1100</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1100</width>
     <height>601</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1100</width>
     <height>601</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1100</width>
     <height>601</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>697</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>送样</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>注样</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>搅拌</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>推条</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>吸样</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>清洗</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="butClose">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/return.png</normaloff>:/icon/return.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
